### PR TITLE
Fix Light Mode Coloring

### PR DESCRIPTION
**Pull request by KevinWu098**
## Summary
Schedule Dropdown, Finals Button, and Clear Schedule did not have the correct `isDarkMode()` ternary applied to them to give them proper light mode colors.

## Test Plan
Only styling changes, make sure everything is correctly viewable on light mode

## Issues
Closes #

### CODE REVIEW

File: apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx [hunk 1]
```diff
@@ -170,7 +171,11 @@ class AddedCoursePane extends PureComponent<AddedCoursePaneProps, AddedCoursePan
                         <PopupState variant="popover">
                             {(popupState) => (
                                 <>
-                                    <Button variant="outlined" {...bindTrigger(popupState)}>
+                                    <Button
+                                        variant="outlined"
+                                        color={isDarkMode() ? 'secondary' : 'inherit'}
```
Comment (ap0nia) [inline apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx]:
```suggestion
                                        color="inherit"
```

File: apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx [hunk 2]
```diff
@@ -203,7 +208,7 @@ class AddedCoursePane extends PureComponent<AddedCoursePaneProps, AddedCoursePan
                         <Button
                             className={this.props.classes.clearSchedule}
                             variant="outlined"
-                            color="secondary"
+                            color={isDarkMode() ? 'secondary' : 'inherit'}
```
Comment (ap0nia) [inline apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx]:
```suggestion
                            color="inherit"
```

File: apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx [hunk 3]
```diff
@@ -313,7 +314,7 @@ function CalendarPaneToolbar(props: CalendarPaneToolbarProps) {
                 <SelectSchedulePopover scheduleNames={scheduleNames} />
                 <Tooltip title="Toggle showing finals schedule">
                     <Button
-                        color={showFinalsSchedule ? 'primary' : 'secondary'}
+                        color={showFinalsSchedule ? 'primary' : isDarkMode() ? 'secondary' : 'inherit'}
```
Comment (ap0nia) [inline apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx]:
```suggestion
                        color={showFinalsSchedule ? 'primary' : 'inherit'}
```

File: apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx [hunk 4]
```diff
@@ -188,7 +189,7 @@ function SelectSchedulePopover(props: { scheduleNames: string[] }) {
         <Box>
             <Button
                 size="small"
-                color="secondary"
+                color={isDarkMode() ? 'secondary' : 'inherit'}
```
Comment (ap0nia) [inline apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx]:
```suggestion
                color="inherit"
```

**Review by ap0nia — DISMISSED** [review]
I'm pretty sure inherit covers the light/dark mode compatibility. Apparently "secondary" is very inconsistent with compatibility.

**Review by MinhxNguyen7 — CHANGES_REQUESTED** [review]
The code looks fine, but the outline should be grey like the buttons on the right side. Let me know if you want to go to bed and do this later. I can merge this in the meantime

**Comment by KevinWu098** [comment]
> The code looks fine, but the outline should be grey like the buttons on the right side. Let me know if you want to go to bed and do this later. I can merge this in the meantime

Ah, so I do agree on that point @MinhxNguyen7 , but the main discussion I had with Brian was that using `inherit` made for a much simpler coding experience when working with both modes. 

> I prefer not having to do complex teraries everytime we want to do compatible colors for different modes
> Just adjust the “inherit” color in the theme provider
 -- Brian

Any thoughts? I'm happy to do the ternaries for now, before we clean it up with a theme provider overhaul, but I'm also happy to just keep it as `inherit`.

**Comment by ap0nia** [comment]
I think it's correct to use "inherit" as the default color for buttons when attempting to target light/dark mode compatibility. But by default, it's only black/white to contrast the current mode. I think the most correct solution would be to make the "inherit" the desired color (i.e. gray) in the theme provider and use it consistently.

Because this issue and other ones with theming just sort of exist intrisically in the app, i.e. dark blue colors in dark mode aren't very visible, I was thinking of opening an RFC and a dedicated PR addressing the theming. So I'm personally ok with inconsistent theming atm (since there's already some inconsistencies lol).

**Review by MinhxNguyen7 — APPROVED** [review]
Sounds good. Ship it

**Comment by MinhxNguyen7** [comment]
@KevinWu098 I can't merge because someone needs to commit something before I can approve it for merging.

**Pull request merged**